src/components/Analytics.js

```diff
@@ -0,0 +1,229 @@
+import React, { Component } from "react";
+import { Animated, StyleSheet, View, Dimensions, TouchableOpacity } from "react-native";
+import {
+  PanGestureHandler,
+  NativeViewGestureHandler,
+  State,
+  TapGestureHandler,
+  FlatList
+} from "react-native-gesture-handler";
+import { Ionicons } from "@expo/vector-icons";
+
+import Block from "./primary/Block";
+import Button from "./primary/Button";
+import Text from "./primary/Text";
+import { SIZES, COLORS } from "../utils/theme";
+import { categories } from "../data/index";
+
+const HEADER_HEIGHT = 70;
+const windowHeight = Dimensions.get("window").height;
+const windowWidth = Dimensions.get("window").width;
```

Comment:
This already exists in utils/theme

Reply:
noted, I've seen it and will be committing a corrextion

Reply:
I made adjustments